### DISCUSSION TITLE
Logic component box

### DISCUSSION THREAD

**Discussion by IanMacb** [FakeFishGames/Barotrauma#6468] [Ideas and suggestions]
I propose a new ship item that can be placed on a wall, and when accessed with a screwdriver it would open a GUI widow to place and organize logic components. Optionally I think consuming wires to connect components should not be necessary inside the box. Any component can then be linked to an external connection of the box, which would then function sort of like a customizable microchip.

**Comment by ViktorShield** [comment]
https://github.com/Regalis11/Barotrauma/discussions/4721

**Comment by Regalis11** [comment]
Implemented
